client/src/app/containers/Game/index.jsx

```diff
@@ -0,0 +1,178 @@
+  let scoreChildren = [];
+  let dealerDesc = 'Dealer: ';
+  let phaseDesc = 'Phase: ';
+  let cutCardDiv;
+  let myHandDiv;
+  let oppHandDivs = [];
+  let cribDiv;
+  let playerNamesByID = {};
+  activeGame['players'].forEach(player => {
+    playerNamesByID[player.id] = player.name;
+  });
+
+  for (const [key, val] of Object.entries(activeGame)) {
+    gameResp.push(`${key}: ${val} `);
+    gameResp.push(<br key={`br ${key}`}></br>);
+
+    switch (key) {
+      case 'players':
+        val.forEach((player, index) => {
+          gameDesc += player.name;
+          if (index < val.length - 1) {
+            gameDesc += ', ';
+          }
+        });
+        break;
+      case 'current_scores':
+        let lagScores = activeGame['lag_scores'];
+        let playerColors = activeGame['player_colors'];
+        let teams = {};
+        for (const [playerName, color] of Object.entries(playerColors)) {
+          if (teams[color]) {
+            teams[color] += `, ${playerName}`;
+          } else {
+            teams[color] = `${playerName}`;
+          }
+        }
+        for (const [color, curscore] of Object.entries(val)) {
+          let team = ` (${teams[color]})`;
+          let scoreStr = `${color}${team}: ${curscore}`;
+          if (lagScores && lagScores[color]) {
+            scoreStr += ` (from ${lagScores[color]})`;
+          }
+          if (color === myColor) {
+            scoreChildren.unshift(
+              <strong key='myTeamScore'>{scoreStr}</strong>,
+              <br key={`br ${color}`}></br>,
+            );
+          } else {
+            scoreChildren.push(scoreStr, <br key={`br ${color}`}></br>);
+          }
+        }
+        break;
+      case 'current_dealer':
+        dealerDesc += val;
+        break;
+      case 'phase':
+        phaseDesc += val;
+        break;
+      case 'cut_card':
+        if (
+          activeGame['phase'] === 'Deal' ||
+          activeGame['phase'] === 'BuildCrib'
+        ) {
+          break;
+        }
+        cutCardDiv = (
+          <div key={'cutCardDiv'}>Cut Card: {jsonCardToCard(val)}</div>
+        );
+        break;
+      case 'hands':
+        for (const [playerID, hand] of Object.entries(val)) {
+          if (!hand) {
+            continue;
+          } else if (playerID === currentUser.id) {
+            myHandDiv = (
+              <div key={'myHandDiv'}>My Hand: {hand.map(jsonCardToCard)}</div>
+            );
+          } else {
+            oppHandDivs.push(
+              <div key={`oppHand ${playerID}`}>
+                {playerNamesByID[playerID]}'s Hand:{' '}
+                {hand.length > 0 ? hand.map(jsonCardToCard) : 'empty/unknown'}
+              </div>,
+            );
+          }
+        }
+
+        break;
+      case 'crib':
+        cribDiv = (
+          <div key={'cribDiv'}>Crib: {val.map(jsonCardToCard).join(', ')}</div>
+        );
+
+        break;
+    }
+  }
+
+  return (
+    <div>
+      <div key={'gameDescDiv'}>{gameDesc}</div>
```

Comment:
1. Shouldn't need `key`s when not rendering iteratively
2. Could simplify code above here `<div>Players are: {gameDesc}</div>`